File: 1.FDP/2° Competencia/C++/Basicos/Ejercicio1.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	//
	cout << "Calcular el cambio de una compra.\n";
	double Total, Pago, Cambio, Totalcambio;
	cout << "Total : ";
	cin >> Total;
	cout << "Pago : ";
	cin >> Pago;
	Cambio= Pago-Total;
	cout << "Cambio = " << Cambio << "\n";

	Totalcambio= Cambio++;

	cout << Totalcambio;
	//
	return 0;
}

File: 1.FDP/2° Competencia/Java/Basicos/Ejercicio1.java
import java.util.Scanner;	

public class Ejercicio1 {
	public static void main(String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular el cambio de una compra.\n");
		double Total, Pago, Cambio;
		System.out.println("Total de compra: ");
		Total= scan.nextDouble();
		System.out.println("Cantidad de pago: ");
		Pago= scan.nextDouble();
		Cambio= Pago-Total;
		System.out.println("Cambio= " + Cambio);
		//
	}
}